gis4wrf/metadata.txt

```diff
@@ -1,6 +1,6 @@
 [general]
 name=GIS4WRF
-version=0.13.2
+version=0.14.0
```

Comment:
I would not bump the version here. I would do that in a different PR when 14.0 is ready and all the changes have been added.
```suggestion
version=0.13.2
```